src/locales/en.json

```diff
@@ -764,6 +812,8 @@
     "aws_title": "Track your Amazon Web Services spending!",
     "azure_desc": "Add an Microsoft Azure account to see a total cost breakdown of your spend by accounts, services, regions, or tags.",
     "azure_title": "Track your Microsoft Azure spending!",
+    "gcp_desc": "Add an Google Cloud Platform account to see a total cost breakdown of your spend by accounts, organizational units, services, regions, or tags.",
```

Comment:
You can remove the "organizational units" text

Reply:
Done